.circleci/config.yml

```diff
@@ -276,6 +276,10 @@ jobs:
       - run:
           name: Install deps
           command: pipenv --python 3.7 install
+      - run:
+          name: Install dev deps
+          # we need to run moto s3 mock server, we start is as a separate process, so it's in the dev deps
```

Comment:
I think it is ok to keep it in the main deps. Since our test runner is entirely about tests I think only misc/linting packages might be considered as development ones. (I think we can merge with that, and I'll place my poetry transition PR on top and fix it there)

Reply:
I've found poetry lockfile changes more reliable compared to pipenv, so I'd keep it this way and wait for your PR to rebase, thank you :)